python/seldon_core/seldon_methods.py

```diff
@@ -102,6 +102,7 @@ def predict(
         if hasattr(user_model, "predict_raw"):
             try:
                 response = user_model.predict_raw(request)
+                client_custom_metrics(user_model, seldon_metrics, PREDICT_METRIC_METHOD_TAG, [])
```

Comment:
As it is, this would double count metrics on both `client_custom_metrics` and `handle_raw_custom_metrics`.